Makefile

```diff
@@ -37,6 +37,9 @@ test_commands:
 	cargo run --bin html5-parser-test >/dev/null
 	cargo run --bin parser_test >/dev/null
 
+authors: ## Update the authors file
+	git log --format='%aN <%aE>' | LC_ALL=C.UTF-8 sort -uf  | grep -v @localhost | grep -v 'dependabot\[bot\]' > AUTHORS
```

Comment:
I figured this was automated.  I wonder whether we can keep the updating of the file manual for now?  Just a thought.  People might not want their email addresses in such an easy place for bots to crawl for sending out spam.

Don't block on this question.  What you have is fine.  Just wanted to ask it.

Reply:
I wouldn't mind to change it to manual. It's a fair point not to have their email directly in the open.

I would suggest to manually create the AUTHORS file, and have some contribution page where people can add their email if they want to the AUTHORS list manually..

Reply:
I've removed the automatic process